spirv_msl.cpp

```diff
@@ -1851,10 +1837,10 @@ void CompilerMSL::emit_glsl_op(uint32_t result_type, uint32_t id, uint32_t eop,
 		break;
 	}
 
-		// TODO:
-		//        GLSLstd450InterpolateAtCentroid (centroid_no_perspective qualifier)
-		//        GLSLstd450InterpolateAtSample (sample_no_perspective qualifier)
-		//        GLSLstd450InterpolateAtOffset
+	// TODO:
```

Comment:
Is this with clang-format 5.0?

Reply:
Here's what I'm using on macOS 10.13.1...

    SPIRV-Cross $ clang-format -version
    clang-format version 5.0.0 (tags/google/stable/2017-06-22)

Reply:
:\ Oh well, guess the number of conflicts is much lower now at least.

Reply:
The formatting effect is to bounce some // comments in all files back and forth by one indentation level.

Surely this can be fixed in `clang-format` configuration or the `format_all.sh` script, can it not?

Otherwise, this is going to be distracting, as it usually affects a number of files...even those that have not been edited. In this PR, I deliberately removed the other files that were ONLY changed by this reformatting, but we can't do that every time.

Reply:
You can try // clang-format off, and // clang-format on, that should block clang-format from doing anything with the section.

Reply:
I can do that for this one. But these kind of comments are peppered around all the files. If I run `format_all.sh`...it makes similar changes to almost all the files.

I think we need to find a configurational way of making this change.